cobra/test/test_core/test_configuration.py

```diff
@@ -32,3 +32,31 @@ def test_solver():
     assert interface_to_str(config.solver) == "glpk_exact"
     # Restore default solver.
     config.solver = "glpk"
+
+
+def test_default_tolerances():
+    """Verify the default solver tolerances."""
```

Comment:
I wouldn't expect the default tolerance to always be the same. Rather, we should implement a default cobrapy tolerance across solvers.

Reply:
I thought that GLPK would always initialize with the same default tolerance.

Reply:
The same version probably does but users might have different GLPK versions. Having a cobrapy specific value definitely makes sense.

Reply:
Does `1E-06` as the default cobrapy solver tolerance sound good?

Reply:
I know of a few models that are feasible with a tolerance of 1E-06 but infeasible with something stricter or the exact solver. So I'm tending towards a stricter default tolerance. I'm not sure how to choose the right one, though.

Reply:
Okay then `1E-09` seems good for the default config.

Reply:
> I know of a few models that are feasible with a tolerance of 1E-06 but infeasible with something stricter or the exact solver. So I'm tending towards a stricter default tolerance. I'm not sure how to choose the right one, though.

This might lead to serious performance regressions though. I would recommend to check with an FVA what impact that might have.